docs/source/options/build_vocab.rst

```diff
@@ -11,7 +11,3 @@ Build Vocab
 
     Transform/SwitchOut : @before
         This transform will not take effet when building vocabulary.
-
-    Transform/Subword/Common : @before
```

Comment:
Particular reason for removing this one but not the other?
(I don"t remember the specifics of the structure here.)

Reply:
The two others are only for `build_vocab`, while the message for `Transform/Subword/Common` should appear for both `build_vocab` and `train` options.